src/components/careers/OpenSourceMission.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+const OpenSourceMission = () => (
+  <div>
+    <p>
+      We are looking for an individual with proven leadership skills and a passion for open-source
+      to lead our open-source efforts.
+    </p>
+    <p>
+      Open-source is vital to Roadie.{' '}
+      <a href="https://backstage.io" target="_blank" rel="noopener noreferrer">
```

Comment:
If you use the `<Link />` component in `src/components/Link` you automatically get "open in new tab" behavior for external links and Google Analytics can track links which lead off the site.

Reply:
I think this would improve the experience, I will add the <Link /> component